app/modules/net.c

```diff
@@ -735,6 +735,14 @@ int net_getaddr( lua_State *L ) {
   lua_pushstring(L, addr_str);
   return 2;
 }
+#if 0
```

Comment:
Tidy up various added debug things, please.

Reply:
Sorry but I think that we should leave this one in.  The GC is net is still not robust and this debug diagnostic can be really useful to sort out what is going on here.  For now this is all commented out, but maybe we should leave it like this IMO

Reply:
Alright, if there's a reasonable justification for it it's all good.